apps/docs/pages/rest-api.mdx

```diff
@@ -7,7 +7,7 @@ The CloudTerms REST API allows you to interact with your applications terms and
 
 
 ```http
-https://api.cloudterms.dev
+https://cloudterms.dev/api/v1/terms
```

Comment:
This base url should just be `https://cloudterms.dev`. The paths are specified below in the docs